### ISSUE TITLE
Slashdot scroll issue

### ISSUE THREAD

**Issue by jcea** [MrAlex94/Waterfox#3032]
### What happened?

Just go to https://slashdot.org/, click on any article, scroll down to the comments section, click in the general area of any comment and you will not able to scroll again in that page until you click on the thin grey border.

This is working correctly under firefox 112.0.2.

### Reproducible?

- [X] I have tested this on Firefox.

### Version

G5

### What platform are you seeing the problem on?

Linux

**Comment by lazymonkey2** [comment]
not happening here.
does it still happens in a new waterfox profile?

**Comment by jcea** [comment]
Yes, I do, with last Waterfox just released (G5.1.6). Reproduction is a bit more tricky that described.

Here you have a reproducible example, fully tested with a clean Waterfox profile:

1. Go to https://news.slashdot.org/story/23/05/14/2316214/
2. Scroll down with the mouse wheel until the first comment in in the middle of the screen.
3. Click with the mouse in any word in that comment
4. Try to scroll now with your mouse wheel, cursors, pageup/down keys, etc. 

I don't think it is relevant, but my Operative System is KUbuntu.
